portfolio/src/main/webapp/index.html

```diff
@@ -27,6 +27,11 @@ <h2>Comments</h2>
         <br>
         <input id="comment-submit" type="submit" />
       </form>
+      <form action="/data" method="POST">
+        <label for="comment-max">Max Comments Shown (between 1 and 50):</label>
+        <input type="number" id="comment-max" name="comment-max" min="1" max="50">
+        <input id="comment-max-submit" type="submit" />
```

Comment:
You now need to check the value in the submit box (as you do for the comment box in checkPostComment()), since currently you can click Submit when there isn't a value and it crashes the site.

Reply:
Ah, thanks! -- I did this with the text box, but I forgot for the max comments shown.